guides/concepts/transactions.md

```diff
@@ -15,7 +15,7 @@ Each transaction has the following attributes:
 > Each transaction sets a [fee](./fees.md#transaction-fee) that is paid by the source account. If this fee is below the network minimum the transaction will fail. The more operations in the transaction, the greater the required fee.
 >
 > #### Sequence number
-> Each transaction has a sequence number. Transactions follow a strict ordering rule when it comes to processing of transactions per account. For the transaction to be valid, the sequence number must match the one stored in the source [account entry](./accounts.md) when the transaction is applied. After the transaction is applied, the source account's stored sequence number is incremented by 1.
+> Each transaction has a sequence number. Transactions follow a strict ordering rule when it comes to processing of transactions per account. For the transaction to be valid, the sequence number must 1 greater than the sequence number stored in the source [account entry](./accounts.md) when the transaction is applied. After the transaction is applied, the source account's stored sequence number is incremented by 1. If the sequence number on the account is 4, then the incoming transaction should have a sequence number of 5. After the transaction is applied, the sequence number on the account is bumped to 5.
```

Comment:
updated to reflect the correct behavior as you described it @bartekn 
sequence number on the account should be the sequence number on the next incoming transaction.

Reply:
Missing `be`:
> For the transaction to be valid, the sequence number must **be** 1 greater than the sequence number stored in the source

Other than that 👍

Reply:
oops, thanks!